.github/ISSUE_TEMPLATE/challenge_proposal.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 🚀 Challenge Proposal
+about: I want to add a new challenge!
+title: ''
+labels: challenge
+assignees: ''
+
+---
+
+## Describe the challenge you'd like to add
+A clear and concise description of what you want to happen. Add any considered drawbacks.
+
+## Suggest a name
+The name of your baby. Consider that there might be adjustments in the challenge name.
+
+## Describe alternatives you've considered
+If relevant, please add different implementation approaches you've considered. Maybe using a different pattern, architecture or library.
```

Comment:
I think you can it should be
`please add different implementations or approaches you've considered.`